### PR TITLE
Groundwork for saving schedule data via API

### PR DESCRIPTION
**Pull request by bamcdole**
This won't be finished until @JeremyWinkler and I test integration with the new API, so I am largely looking for in-progress feedback at this point.

### CODE REVIEW

**Comment by lweber-d2l** [comment]
Looks good so far 👍

File: src/api/routes.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ const DS_API_VERSION = '1.28';
 
 export class Routes {
 	static DataSets() { return `/d2l/api/lp/${DS_API_VERSION}/dataExport/list`; }
-	static GetSchedule(scheduleId) { return `/d2l/api/customization/${API_VERSION}/ads/scheduler/schedules/${scheduleId}`; }
+	static ExistingSchedule(scheduleId) { return `/d2l/api/customization/${API_VERSION}/ads/scheduler/schedules/${scheduleId}`; }
+	static NewSchedule() { return `/d2l/api/customization/${API_VERSION}/ads/scheduler/schedules`; }
```
Comment (IhabAlkabary) [inline src/api/routes.js:7]: why the need for `schedules` at the end of this route

Comment (bamcdole) [reply src/api/routes.js:7]: @JeremyWinkler Maybe you can answer this one

Comment (IhabAlkabary) [reply src/api/routes.js:7]: I've checked existing patterns in Valence, and it the same, so no issue here

Comment (JeremyWinkler) [reply src/api/routes.js:7]: Just following things in this article (https://medium.com/hashmapinc/rest-good-practices-for-api-design-881439796dc9), our API base route is `ads/scheduler` and the resource we are trying to access is `schedules`. Then for dealing with a specific schedule (updating or getting), we use `schedules/{scheduleId}`. Trying to somewhat follow this image:
image

**Review by lweber-d2l — APPROVED** [review]
This is looking good 👍

**Pull request merged**